netlify/landings/studio.yaml

```
@@ -0,0 +1,101 @@
+path: /studio
+title: Stoplight Studio
+color: blue
+relatedTags:
+  - blog-design
+hero:
+  image:
+    alt: Stoplight Studio
+    src: /images/design_header.png
+  subtitle: >-
+    Build first-class APIs with a collaborative, powerful, visual editor that
+    allows you to prototype and share your API within minutes
+  title: 'Stoplight Studio'
+  ctas:
+    - color: green
+      href: ''
+      name: Download Now
+    - color: green
+      href: ''
+      name: Sign Up Now      
+featureSection:
+  actionBar:
+    buttons:
+      - href: 'https://next.stoplight.io/'
+        title: Get Started
+    enabled: true
+    text: Ready to Model and Design?
+  title: OpenAPI v2/v3 Visual Editor
+  description: >-
+    Start designing your API in minutes. Use the [OpenAPI
+    Specification](/api-design-guide/oas-spec/) (formerly known as Swagger) with
```

```suggestion
    Specification](/api-design-guide/openapi/) (formerly known as Swagger) with
```

This is changing in #1038

---

Can we change this to "Using OpenAPI (formerly known as Swagger)" because the specification is a technical implementation detail that the user doesn't care about. We don't talk about JSON Specification or even Swagger Specification but we always talk about OpenAPI Specification for some reason. :)

---

Can you update this in #1038 ?